rotkehlchen/globaldb/schema.py

```diff
@@ -171,6 +171,16 @@
 );
 """
 
+DB_CREATE_REQUESTS_CACHE = """
+CREATE TABLE IF NOT EXISTS requests_cache (
+    type CHAR(1) NOT NULL,
+    name TEXT NOT NULL,
+    value TEXT,
```

Comment:
Not sure but maybe it would be better to have `BLOB` `value` to have more flexibility?

Reply:
Hm, yeah, okay, was just looking for different opinions :)